Arion/include/Arion/SimpleShapeIntersection.hpp

```diff
@@ -100,6 +99,33 @@ std::pair<glm::dvec3, glm::dvec3> CalculateContactPoints(SimpleShape const* a, S
 template <typename ShapeA, typename ShapeB>
 double CalculatePenetration(SimpleShape const* a, SimpleShape const* b, CacheBase* cache);
 
+/**
+ * @brief Calculates contact manifold for given shapes
+ *
+ * @attention Must be called strictly after the corresponding CalculateIntersection function call, otherwise result is undefined
```

Comment:
right now both `CalculateContactNormal` and `CalculateContactManifold` have to be called `strictly after the corresponding CalculateIntersection function call`
does `strictly` apply?